### PR TITLE
Hamburger Menu Restyling

### PR DESCRIPTION
**Pull request by kennethlien**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Added: 
- User's name
- Report Issue button
- Logout button
- Restyled hamburger button
- New file: styled/hamburger.js

Removed:
- News Screen option (Files still there, to be deleted in future)
- Inline styling for Hamburger.js

## Relevant Links
https://flexboxfroggy.com/

### Online sources
https://flexboxfroggy.com/

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs
https://github.com/calblueprint/dccentralkitchen/commit/a20a3a0f338234f0792eefe6204e89500771ad1b

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

Click around for bugs!
[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

- Update Report Issue to jump to actual link from AirTable
- Reduce padding between the clickable tabs (Stores/Points History/Resources), could not figure out 
   where to edit that
- Change top value of styled/hamburger.js to reflect Figma (currently looks weird on iPhone X and newer

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
- Yes

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"
Simulator Screen Shot - iPhone 11 Pro - 2020-03-18 at 19 03 24
Simulator Screen Shot - iPhone 11 Pro - 2020-03-18 at 19 03 30

CC: @tommypoa  @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: components/Hamburger.js [hunk 1]
```diff
@@ -1,38 +1,17 @@
 import React from 'react';
-import {
-    TouchableOpacity,
-    Button
-} from 'react-native';
-
+import { TouchableOpacity, Button } from 'react-native';
+import { FontAwesome5 } from '@expo/vector-icons';
+import { HamburgerButton } from '../styled/hamburger';
+//TODO: ADD FONTAWESOME ICON
```
Comment (wangannie) [inline components/Hamburger.js]: ??

File: styled/hamburger.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+//TODO: Fix top to match figma once we figure out iPhone alignment
+export const HamburgerButton = styled(TouchableOpacity)`
+  background-color: #fff;
+  width: 40px;
+  height: 40px;
+  zindex: 100;
+  position: absolute;
+  top: 63px;
+  left: 16px;
```
Comment (wangannie) [inline styled/hamburger.js]: I think you can solve your positioning issues by moving <Hamburger navigation={this.props.navigation} /> into the MapView in `MapScreen.js`, right above SearchBar. Then you can make the top here the same as the top margin on SearchBar (20px I believe), and it should stay lined up with the height of the search bar, which has mostly been looking fine on different kinds of screens.

File: styled/hamburger.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  border-radius: 23px;
+  border-width: 4px;
+  border-color: #fff;
```
[the remark this answers is not in the document]
Comment (wangannie) [reply styled/hamburger.js]: Remove `border-width` and `border-color`

File: navigation/AppNavigator.js [hunk 4]
```diff
@@ -38,33 +48,117 @@ class AuthLoadingScreen extends React.Component {
...
+
+        <DrawerItems {...this.props} />
+
+        <View
+          style={{
+            flex: 1,
+            flexDirection: 'column',
+            justifyContent: 'flex-end',
+            verticalAlign: 'bottom'
+          }}>
```
[the remark this answers is not in the document]
Comment (kennethlien) [reply navigation/AppNavigator.js]: god bless flexbox froggy

File: components/Hamburger.js (hunk 1 above)
Comment (kennethlien) [reply components/Hamburger.js]: oops, this was a leftover comment
